drivers/include/at.h

```diff
@@ -448,10 +486,12 @@ int at_parse_resp(at_dev_t *dev, char const *resp);
 /**
  * @brief   Read a line from device
  *
+ * Stops at the first DCE EOL sequence.
+ *
  * @param[in]   dev         device to operate on
  * @param[in]   resp_buf    buffer to store line
  * @param[in]   len         size of @p resp_buf
- * @param[in]   keep_eol    true to keep the CR character in the response
+ * @param[in]   keep_eol    true to keep the trailing EOL sequence in the response
```

Comment:
This used to keep only the first character in the EOL sequence whenever `keep_eol` was true, which I find kind of arbitrary and creates a lot of edge cases to think about. Now you can either have the whole trailing EOL sequence or nothing. If e.g. the EOL is composite (`\r\n`), individual EOL characters will always be kept.